packages/console/src/components/Uploader/FileUploader/index.tsx

```diff
@@ -2,7 +2,7 @@ import type { AllowedUploadMimeType, UserAssets } from '@logto/schemas';
 import { maxUploadFileSize } from '@logto/schemas';
 import classNames from 'classnames';
 import { useCallback, useEffect, useState } from 'react';
-import { useDropzone } from 'react-dropzone';
+import { type FileRejection, useDropzone } from 'react-dropzone';
```

Comment:
import type ?

Reply:
It is import type ?

<img width="571" alt="image" src="https://user-images.githubusercontent.com/10806653/229817884-33489226-754e-42bc-955a-0cae29d00c0d.png">

https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/